### ISSUE TITLE
per user storage

### ISSUE THREAD

**Issue by fantapop** [danielzen/ionic-pouchdb-todo#3]
Hey There,

The set up that you have here is pretty neat.  I'm trying to wrap my head around how it would be used in an application.  In general, if I'm connecting to the same instance of couch for the sync and I choose the same database name, each user of the application would share the same todo list right?  What kind of a pattern would you use for having per user data storage?

In my exact case, I'm writing an application that has some data I'm currently keeping in localstorage.  I thought it would be a good idea to back it up somewhere off the phone as well.   Your libraries seemed like a good option.

**Comment by danielzen** [comment]
Yes, this was a spike turned into a library. I have not yet really attacked the following 3 issues:
* Multi-user Authentication
* Conflict Resolution
* Ordered Data

All of  which need some attention. You could give each user their own database, but I'm not sure that would be the most efficient way to proceed. Please feel free to share your thoughts!
